source/mapping-rules/fields/field-customizations.txt

```diff
@@ -0,0 +1,102 @@
+.. _rm-field-customizations:
+
+======
+Fields
```

Comment:
[suggestion] Is this a concept? Should we title it "About Fields" or even go off-script and call it "Customize Fields"?